TFproviderTest.yml

```diff
@@ -63,12 +63,12 @@ pipelines:
             - helm repo update
             - add_run_variables rt_helm_chart_version=$(helm search repo | grep "${ARTIFACTORY_DISTRIBUTION} " | awk '{$1=$1};1' |  cut -f2 -d " ")
             - echo "Helm chart version "${rt_helm_chart_version}
-            - add_run_variables artifactory_version=$(helm search repo | grep "${ARTIFACTORY_DISTRIBUTION} " | awk '{$1=$1};1' |  cut -f3 -d " ")
-            - echo "Artifactory version "${artifactory_version}
+            - add_run_variables ARTIFACTORY_VERSION=$(helm search repo | grep "${ARTIFACTORY_DISTRIBUTION} " | awk '{$1=$1};1' |  cut -f3 -d " ")
+            - echo "Artifactory version "${ARTIFACTORY_VERSION}
             - add_run_variables xray_helm_chart_version=$(helm search repo | grep "/xray" | awk '{$1=$1};1' |  cut -f2 -d " ")
             - echo "Xray Helm chart version "${xray_helm_chart_version}
-            - add_run_variables xray_version=$(helm search repo | grep "/xray" | awk '{$1=$1};1' |  cut -f3 -d " ")
-            - echo "Xray version "${xray_version}
+            - add_run_variables XRAY_VERSION=$(helm search repo | grep "/xray" | awk '{$1=$1};1' |  cut -f3 -d " ")
```

Comment:
What format does this take?

Reply:
The requirements came from [Slack documentation](&amp)

Reply:
it's just text format, if I understood your question correct